Support Native Protected Mode

The [instruction](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/docs/platforms/x86_64_linux/x86_64_native.md#how-to-build) describes how to build Edge Orchestration as 'x86_64ns'.
Do we support it? 

```
$ make create_context CONFIGFILE=x86_64ns
cp: cannot stat 'configs/defconfigs/x86_64ns': No such file or directory
make: *** [Makefile:269: .config] Error 1
```

--- a/configs/defconfigs/x86_64ns
+++ b/configs/defconfigs/x86_64ns
@@ -1,0 +1,14 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Edge-Home-Orchstration Configuration
+#
+CONFIG_CONFIGFILE="x86_64ns"
+CONFIG_X86_64=y
+# CONFIG_X86 is not set
+# CONFIG_ARM is not set
+# CONFIG_ARM64 is not set
+# CONFIG_CONTAINER is not set
+CONFIG_NATIVE=y
+# CONFIG_ANDROID is not set
+# CONFIG_MNEDC is not set
+CONFIG_SECURE_MODE=y

